pandas/_libs/src/parser/tokenizer.c

```diff
@@ -600,7 +599,7 @@ static int parser_buffer_bytes(parser_t *self, size_t nbytes) {
         TRACE(("PUSH_CHAR: ERROR!!! slen(%d) >= stream_cap(%d)\n", slen,      \
                self->stream_cap))                                             \
         int64_t bufsize = 100;                                                \
-        self->error_msg = (char *)malloc(bufsize);                            \
+        self->error_msg = malloc(bufsize);                                    \
```

Comment:
extra whitespace

Reply:
This is by design to keep all of the slashes right aligned